docs/tutorial/lambda-aws.txt

```diff
@@ -0,0 +1,121 @@
+============================================
+Deploy to Serverless AWS Lambda with BrefPHP
+============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. versionadded:: 1.17
+
+Overview
+--------
+
+`Bref <https://bref.sh>` allows to deploy serverless PHP applications on AWS Lambda.
+In this tutorial, you will deploy a simple PHP application with the MongoDB PHP extension,
+and connect to an Atlas cluster using AWS IAM authentication.
+
+Pre-requisites
+--------------
+
+Before you begin, you must install bref on your machine. You can follow the
```

Comment:
```suggestion
Before you begin, you must install Bref on your machine. You can follow the
```

Bref's own docs capitalize the name, so I think it makes sense to do so throughout our own docs.